pyecharts_snapshot/main.py

```diff
@@ -13,36 +14,41 @@
 else:
     from io import BytesIO
 
-PHANTOMJS_EXE = "phantomjs"
+PHANTOMJS_EXEC = "phantomjs"
 NOT_SUPPORTED_FILE_TYPE = "Do not support file type %s"
 DEFAULT_DELAY = 1.5
 
 
 def main():
     if len(sys.argv) < 2 or len(sys.argv) > 4:
-        raise sys.exit(-1)
-
-    try:
-        does_phantomjs_exist()
-    except OSError:
-        print("No phantomjs found in your path. Please install it!")
-        sys.exit(-1)
-
+        if len(sys.argv) == 2:
+            webbrowser.open('https://github.com/pyecharts/pyecharts-snapshot')
```

Comment:
better not invite a webbrowser here. if you want to, you can put the link in the help text..